perception/tensorrt_yolox/launch/yolox_sPlus_opt.launch.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
```

Comment:
[imo]
Could you change this filename to `yolox_s_plus_opt.launch.xml` or `yolox_s_opt.launch.xml` to ensure consistency with other file names (snake_case).

Reply:
@miursh 
Thank you for your comment. Since I agree with your opinion, I changed the filename as you suggested in 8f6d34523917f10f95e06c6b36d3254e35166b9a. Thank you!